app/Http/Controllers/DecisionController.php

```diff
@@ -28,15 +35,24 @@ public function index(): Factory|View|Application
         ]);
     }
 
+    public function showDecisionPage(): Response
```

Comment:
Pls stick to naming convention in Laravel -> Index would be sufficient, because show is used for single Items

Reply:
But it returns only **one** site